src/classes/SuperchainFormFrame.ts

```diff
@@ -85,12 +85,11 @@ export class SuperChainFormFrame {
   };
 
   static getWebsiteUrl = (url: string) => {
-    const regex =
-      // eslint-disable-next-line no-useless-escape
-      /^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,})([\/\w \.-]*)*\/?$/;
-    if (!regex.test(url)) {
-      throw new Error("Invalid URL");
+    if (!url.startsWith("http://") && !url.startsWith("https://")) {
+      return `https://${url}`;
```

Comment:
you're not validating the URL here, should append and then call new URL with the new value